crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -36,11 +36,14 @@ pub struct JobCancelOpts {
 
 #[derive(Parser)]
 pub struct JobCatOpts {
+    /// Select job
     pub job_id: u32,
 
+    /// Select task(s) outputs to view
     pub task_selector_arg: SelectorArg,
 
     #[clap(long)]
+    /// Type of output(stdout, stderr) to display
```

Comment:
```suggestion
    /// Type of output to display
    #[clap(possible_values = &["stdout", "stderr"])]
```
But as mentioned earlier, I would prefer this to be a subcommand, i.e. `$ hq job cat [stdout/stderr] <cat-options>`.